.github/workflows/stale.yaml

```diff
@@ -0,0 +1,28 @@
+name: Stale issues and pull requests
```

Comment:
Question: What will happen with all open issues / PRs on the first invocation of this job? Will all issues / PRs with >90 days of inactivity get labelled `life-cycle/stale`? Nothing will get closed, right?

Reply:
Yup, it will be a busy first day for this action

Reply:
Yes to which part of the question? Yes to both? Meaning the stale label will be added to all issues older than 90 days, but NO rotten labels will be added on the initial run, right?

As far, as I understand it, an issue first gets the stale label after 90 days of inactivity. It then needs to sit with the stale label for 90 days before it is relabeled as rotten (pinging all watchers of the issue in the process). It then needs to sit with the rotten label for 90 days before it is automatically closed.

If you remove either label (regardless of which lifecycle state it’s in), the whole process starts over and you have another 270 days before the issue is closed (90 until stale label, another 90 until rotten, and another 90 until closed).

So as long as the initial run only applies the stale label, we have 180 more days until an auto close.

Reply:
Actually, here’s the rules applied by k8s, can we just configure ours the same:
https://github.com/kubernetes/website/issues/34183#issuecomment-1242765483

Reply:
after seeing the action, looks like issues older than 180 days will get closed

Reply:
>So 90,30,30 instead of 90,90,90 as I said above.

This GitHub action only has 2 stages, so it would be 90-60, is that ok? , this PR proposes 90-90

Reply:
I have updated the PR
removed the rotten label
and adjusted the closing time

Reply:
@klueska @elezar @cdesiniotis